### DISCUSSION TITLE
Set labels only for PRs with failing tests?

### DISCUSSION THREAD

**Discussion by ChristianFroestl9781** [renovatebot/renovate#16977] [Request Help] [answered]
Lovely renovate community,

we have configured renovate to open PRs if the tests for an auto merge are failing (automergeRequireAllStatusChecks ) and we would like to receive notification in our alerting channel for this.
Therefore, we would like to use the MS teams GitHub integration which offers the possibility to subscribe to GitHub changes based on labels. 
Renovate is supporting labels as well, but I cannot find any way to differentiate labels for different results. Example: label:passed for all PRs which passed the tests and label:failed for unsuccessful tests.
Is it possible to set custom labels in renovate only for PRs where tests have not passed? Or do you have an idea of another workaround how to get notified only for PRs with failed tests? 

Thanks,
Christian

**Comment by Gabriel-Ladzaretti** [comment, marked as answer]
Unfortunately it seems that's not possible as of now, labels are either set globally or as part of a package rule. The later corresponds to packages/dependencies only.

https://docs.renovatebot.com/configuration-options/#labels
https://docs.renovatebot.com/configuration-options/#packagerules

**Reply by Gabriel-Ladzaretti** [reply]
This possibly can be converted to a feature request.
